README.rst

```diff
@@ -7,13 +7,23 @@ This repository contains the Pine64 fork of Bouffalo Lab's SDK for their BL602
 Wi-Fi/BLE Combo RISC-V SoC. Documentation, including upstream documentation,
 translations, mirrored technical documentation, and original reverse engineered
 documentation is at `bl602-docs <https://github.com/pine64/bl602-docs>`_, which
-is cloned as a submodule of this repository at ``docs``. Use `git submodule update --init` to initialize the submodule. 
+is cloned as a submodule of this repository at ``docs``. Use
+`git submodule update --init` to initialize the submodule.
 
 This repository is the central focus of Pine64's
 `Nutcracker Challenge <https://www.pine64.org/2020/10/28/nutcracker-challenge-blob-free-wifi-ble/>`_.
 
 Quick Start
 -----------
+Firstly, you should clone the repository and update the submodules (for
+documentation and building)::
+
+    git clone git@github.com:pine64/bl_iot_sdk.git
+    git submodule update --init
```

Comment:
Unnecessary SSH authentication, normal people don't have push rights. Also, `--recursive` clones exist, `git clone --recursive https://github.com/pine64/bl_iot_sdk`

Reply:
Good catch @WildCryptoFox ! @danielbarry please can you solve those issues? thanks

Reply:
Yeah, this and the second one should be changed to https:// because of the previously mentioned reasons.

Reply:
At least now the lack of `--recursive` isn't an "issue", just more succinct. OTOH people often clone naively without it, so it makes sense to keep at least a suggestion to use `git submodule update --init [--recursive]` to correct ordinary clones.

Reply:
Have an update ready for this in a commit :)